.github/renovate.json

```diff
@@ -59,7 +59,7 @@
         "autoprefixer",
         "react-router-dom",
         "org.liquibase:liquibase-core",
-        "io.vertx:vertx-json-schema",
+        "com.networknt:json-schema-validator",
```

Comment:
Is this change necessary? We only exclude vertx-json-schema because it (transitively) upgrades SnakeYAML which we are not ready for.